README.md

```diff
@@ -133,6 +133,7 @@ Common configurations:
 |tikv.batchsize|128|Request batch size|
 |tikv.async_commit|true|Enalbe async commit or not|
 |tikv.one_pc|true|Enable one phase or not|
+|tikv.apiversion|1|api-version of tikv server|
```

Comment:
Still API V1 ?

Reply:
Default is api v1 and can set it with tikv.apiversion to 2.

Reply:
Apiv2 is an experimental feature in tikv and usually used for rawkv. So v1 is stiil the default version.